app/src/main/java/com/hedvig/app/feature/hanalytics/ExperimentsInitializer.kt

```diff
@@ -0,0 +1,22 @@
+package com.hedvig.app.feature.hanalytics
+
+import android.content.Context
+import androidx.startup.Initializer
+import com.hedvig.app.feature.di.KoinInitializer
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+
+class ExperimentsInitializer : Initializer<Unit>, KoinComponent {
+    private val experimentManager: ExperimentManager by inject()
+
+    override fun create(context: Context) {
+        CoroutineScope(Dispatchers.IO).launch {
+            experimentManager.preloadExperiments()
```

Comment:
What happens if this call fires before a token has been acquired?

Reply:
Removed this Initializer because of exactly this problem, thanks for the heads up!